VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -227,14 +227,33 @@ def get_http_util(self):
         from HttpUtil import HttpUtil
         return HttpUtil(self.logger)
 
+	def get_wireserver_endpoint_uri(self):
+
+		wireserver_endpoint_file = '/var/lib/waagent/WireServerEndpoint'
```

Comment:
would be cleaner to define this in Common.py alongside the wire server IP constant